pluginlib/pluginlib-extras.cmake

```diff
@@ -44,3 +44,10 @@ endif()
 find_package(tinyxml2_vendor REQUIRED)
 find_package(TinyXML2 REQUIRED)
 list(APPEND pluginlib_LIBRARIES ${TinyXML2_LIBRARIES})
```

Comment:
@dirk-thomas do we have to export `tinyxml2` libraries along with `pluginlib`'s now that it is an exported interface?

Reply:
Yes, the modern CMake approach is only added - it doesn't remove the classic CMake variable to maintain backward compatibility.